packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -497,6 +499,32 @@ sdk.server('https://eu.api.example.com/v14');`),
     return sourceFile;
   }
 
+  /**
+   * Create a placeholder `README.md` file in the repository, with information on how to use/administer the SDK.
+   *
+   */
+  createREADME() {
+    const file = `# \`@api/${this.identifier}\`
+
+This SDK was autogenerated by the [\`api\` SDK generator](https://api.readme.dev), powered by [ReadMe](https://readme.com) 🦉
+
+Add SDK setup information and usage examples here so your users get started in a jiffy! 🚀
+
+<!---
+
+Here's some additional info about the generated SDK:
+
+\`api\` version: ${PACKAGE_VERSION}
+Generated at ${new Date().toISOString()}
```

Comment:
After https://github.com/readmeio/api/pull/775 is merged in you should be able to grab the `createdAt` date from the lockfile for this with this code:

```ts
storage.getFromLockfile().createdAt
```

Reply:
should be able to pull this in now that `createdAt` has been added to the lockfile.